src/bereikbaarheid/resources/utils.py

```diff
@@ -108,21 +108,25 @@ def refresh_materialized(db_table: str):
 # -------------------------------------------
 
 
-def convert_to_date(date: str = None, format: str = "%d/%m/%y %H:%M") -> datetime:
-    """Convert string format to datetime"""
+def convert_to_date(date: str = None) -> datetime:
+    """Convert string format %d/%m/%y %H:%M of %Y-%m-%d %H:%M:%S.%f to datetime"""
 
+    formats_allowed = ["%d/%m/%y %H:%M", "%Y-%m-%d %H:%M:%S.%f"]
     try:
-        _date = datetime.datetime.strptime(date, format)
-    except:
-        try:
-            format = "%Y-%m-%d %H:%M:%S.%f"
-            _date = datetime.datetime.strptime(date, format)
-        except:
-            raise ValueError(
-                f"verkeerd datumformat voor {date}, gewenst format is '%d/%m/%y %H:%M' of '%Y-%m-%d %H:%M:%S.%f'"
-            )
+        for format in formats_allowed:
+            try:
+                _date = datetime.datetime.strptime(date, format)
+                break
+            except ValueError:
+                pass
+
```

Comment:
Ik zou de except UnboundLocalError vervangen door een check ` if _date == None: raise ValueError()`. Dan hou je de code platter en is het ook beter leesbaar. Dit is een beetje een php-manier van doen, waar veel isset() wordt gebruikt.